manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+PackageLocale: en-US
+Publisher: HoYoverse
```

```suggestion
Publisher: COGNOSPHERE PTE. LTD.
```
This should match Control Panel